README.md

```diff
@@ -14,50 +26,35 @@ Installing ember-cli-yadda is a breeze. All you need to do is run the following
 ember install ember-cli-yadda
 ```
 
-##### Versions >= 0.2.0
-
 This adds the latest version of yadda to your node modules, along with [ember-browserify](https://www.npmjs.com/package/ember-browserify) (to allow yadda to run in the browser). It also adds the following files:
 
 ```
 /tests/acceptance/steps/steps.js
+/tests/integration/steps/steps.js
 /tests/unit/steps/steps.js
+/tests/helpers/yadda.js
+/tests/helpers/yadda-annotations.js
 ```
 
 You may specify the version of yadda by changing it in package.json and running `npm install`.
 
-##### Versions <= 0.1.0
+## Usage
```

Comment:
Could add upgrade instructions above here.  Something like the below with links to appropriate sections below...

## Upgrading to v0.4.x
1. Upgrade your project to use Ember 3.x.
1. Install the latest version of ember-cli-yadda.
1. Run `ember-cli-yadda g` to add the most recent files from the blueprint to your project.
1. Add the appropriate setup annotation to each Feature or Scenario.
1. Customize `tests/helpers/yadda-annotations.js` with any additional setup logic that is needed.
1. Optional - update your steps files to use async / await.
1. Optional - group your tests into suites by using additional annotations.

Reply:
Thanks for the suggestion! I added this section, with some changes though:

> Upgrade your project to use Ember 3.x.

This is not required, you can still run it with older versions!

Also added some more instructions for actually rewriting your step files to the new testing APIs!